third_party/uapi/xe/xe_drm.h

```diff
@@ -513,6 +520,7 @@ struct drm_xe_query_topology_mask {
 #define DRM_XE_TOPO_DSS_GEOMETRY	(1 << 0)
 #define DRM_XE_TOPO_DSS_COMPUTE		(1 << 1)
 #define DRM_XE_TOPO_EU_PER_DSS		(1 << 2)
```

Comment:
There are other changes in this header that are already available in drm-next. It's probably easier if you just sync the header with drm-next and add on top this change. This way it won't conflict with future updates.
This is what we have in drm-next:
 
```
#define DRM_XE_TOPO_DSS_GEOMETRY        1
#define DRM_XE_TOPO_DSS_COMPUTE         2
#define DRM_XE_TOPO_L3_BANK             3
#define DRM_XE_TOPO_EU_PER_DSS          4
```

Reply:
Also, feel free to commit this once DRM_XE_TOPO_SIMD16_EU_PER_DSS reaches drm-next as per rules in https://docs.kernel.org/gpu/drm-uapi.html:

> The kernel patch can only be merged after all the above requirements are met, but it must be merged to either drm-next or drm-misc-next before the userspace patches land. uAPI always flows from the kernel, doing things the other way round risks divergence of the uAPI definitions and header files.